lib/ddtrace/contrib/elasticsearch/core.rb

```diff
@@ -0,0 +1,61 @@
+require 'uri'
+require 'ddtrace/pin'
+require 'ddtrace/ext/app_types'
+
+URL = 'elasticsearch.url'.freeze
+METHOD = 'elasticsearch.method'.freeze
+TOOK = 'elasticsearch.took'.freeze
+PARAMS = 'elasticsearch.params'.freeze
+BODY = 'elasticsearch.body'.freeze
+
+DEFAULTSERVICE = 'elasticsearch'.freeze
+
+module Datadog
```

Comment:
Do we want to use a DatadogTrace or similar toplevel module so we don't have to worry about conflicts with other datadog libs?